spec/features/resource_manager_manages_holidays_spec.rb

```diff
@@ -92,64 +119,112 @@ def when_they_view_holidays
   end
 
   def then_they_can_see_the_holidays_for_this_week
-    expect(@page.all_events).to eq [
-      {
-        title: 'First user holiday - First Guider',
-        time: '9:00 - 11:00'
-      },
-      {
-        title: 'Merged Holiday - First Guider, Second Guider',
-        time: '13:00 - 14:00'
-      }
-    ]
+    holidays = @page.calendar.holidays
+
+    expect(holidays.count).to eq 2
+
+    expect(holidays.first[:title]).to eq 'First user holiday'
```

Comment:
You could push this into the page object, it gets called in unison frequently enough.